service.py

```diff
@@ -523,6 +523,22 @@ def allowed_file(filename):
 @app.route("/download/<filename>", methods=['GET'])
 def download_file(filename):
     return send_from_directory(os.path.join(app.config['UPLOAD_FOLDER']), filename, as_attachment=True)
+
+# ===========================================================================
+# show pdf file
+# ===========================================================================
+@app.route("/show/<filename>", methods=['GET'])
+def show_file(filename):
+    file_dir = os.path.join(basedir, app.config['UPLOAD_FOLDER'])
+    if request.method == 'GET':
+        if filename is None:
+            pass
+        else:
+            pdf_data = open(os.path.join(file_dir, '%s' % filename), "rb").read()
```

Comment:
@Weichao-Chen 

What if the undesired event that **filename could not be found** happens?

Anyway to exit gracefully?


**Hui**